src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Check.java

```diff
@@ -3002,7 +2968,7 @@ void checkAccessFromSerializableElement(final JCTree tree, boolean isLambda) {
                 isEffectivelyNonPublic(sym)) {
                 if (isLambda) {
                     if (belongsToRestrictedPackage(sym)) {
-                        log.warning(LintCategory.SERIAL, tree.pos(),
+                        log.warning(tree.pos(),
                                     Warnings.AccessToMemberFromSerializableLambda(sym));
```

Comment:
so this one won't be a lint warning anymore I guess

Reply:
Actually I think this is another accidentally omitted conversion from `Warnings` to `LintWarnings`.

Reply:
True, this is a bug in compiler.properties.